### PR TITLE
add script for refreshing redis to be run by cronjob

### PR DESCRIPTION
**Pull request by acharb**
some of the new v2 endpoints (dex, fees, ledger op stats) have data that updates when redis is cleared, so adding the code to do that that will be triggered by a cron job

### CODE REVIEW

File: backend/src/dex/data.ts [hunk 1]
```diff
@@ -71,11 +82,35 @@ export async function getUniqueAssetsData() {
...
+  const activeAccounts = await fetchCachedData(
+    REDIS_DEX_KEYS.DEX_ACTIVE_ACCOUNTS,
+    query,
+  );
+
+  return activeAccounts[0]?.data;
+}
+
+//TODO: Currently unused due to a pending update to how we fetch asset prices. Once they're available we should refactor and make use of the function below again.
+const cache24hKey = "dex-volume-sum-24h";
```
Comment (acharb) [inline backend/src/dex/data.ts:110]: I moved this stuff to the bottom of the file under this TODO comment, confused me we're not using this (yet)

File: backend/src/dex/data.ts [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
 import {
   bigQueryEndpointBase,
   fetchBigQueryData,
   fetchCachedData,
 } from "../utils";
 
-const cache24hKey = "dex-volume-sum-24h";
-const cache48hKey = "dex-volume-sum-48h";
-const cacheOverallKey = "dex-volume-sum-overall";
+export const REDIS_DEX_KEYS = {
```
Comment (acharb) [inline backend/src/dex/data.ts:9]: also moved the redis keys to constants that weren't already

File: backend/package.json [hunk 3]
```diff
@@ -11,7 +11,9 @@
     "start": "UPDATE_DATA=true DEV=true ts-node ./src/app.ts",
     "clean": "rm -rf node_modules && rm -rf build",
     "test": "jest --forceExit",
-    "build": "tsc --project ."
+    "build": "tsc --project .",
+    "redis:reset_day": "ts-node ./src/redis/redisRefresh.ts day",
```
Comment (acharb) [inline backend/package.json:15]: on second look I think the trigger will have to be part of the Dockerfile 🤔  , but I'll change that  in a second PR if needed

Comment (piyalbasu) [reply backend/package.json:15]: I have limited context, but my hunch is that you'll want to trigger this from the Dockerfile

**Review by piyalbasu — APPROVED** [review]

**Comment by acharb** [comment]
@piyalbasu fyi added this small commit https://github.com/stellar/dashboard/pull/310/commits/19b5115efe8ec4eca997b17d70a6b83db1ff06cd
the production command will be ran from Docker/kube but DEV=true is needed when running it locally from package.json

**Pull request merged**